docs/general/builtins/scalar.rst

```diff
@@ -1393,6 +1393,40 @@ See below for an example::
     +-------------+
     SELECT 1 row in set (... sec)
 
+.. _scalar-trunc:
+
+``trunc(number[, precision])``
+------------------------------
+
+Returns `number` truncated to the specified `precision`.
+
+`number` is the ``numeric`` value to be truncated.
+`precision` is an ``integer`` indicating the number of decimal places to
```

Comment:
you can drop this line too. maybe add the bit about "the number of decimal places" to the first para in this section as a clarification, i.e.:

```
Returns ``number`` truncated to the specified ``precision`` (i.e., decimal places).
```